san/tests/test_sanbase_integration.py

```diff
@@ -443,7 +443,6 @@ def test_available_metrics():
     functions_list = [
         'average_token_age_consumed_in_days',
         'burn_rate',
-        'daily_active_addresses',
```

Comment:
?

Reply:
These metrics are no longer available, they're only available from the on-chain list. I thought I've removed them with the last PR, but it turned out, I haven't.